docs/concepts/privacy-and-security.md

```diff
@@ -69,6 +68,14 @@ You can make your Office Add-ins available to the public by publishing them to A
 
 - Supports a user-review system for available add-ins to promote a self-policing community.
 
+## Optional connected experiences
+
+Users can turn off [optional connected experiences in Office](/deployoffice/privacy/optional-connected-experiences) desktop and mobile clients. For Office Add-ins, generally this means that users who turn off the **Optional connected experiences** setting can no longer access add-ins or the Office Store through these clients. However, certain Microsoft add-ins that are considered essential or business-critical, and add-ins deployed by an organization's IT admin are still available. Also, the setting doesn't affect the availability of add-ins and the Store in Office on the web.
```

Comment:
generally? lets be more bold. The impact of turning OFF Optional connected experiences means.....